source/helpers/selector-observer.tsx

```diff
@@ -36,7 +36,7 @@ const registerAnimation = onetime((): void => {
 
 export default function observe<
 	Selector extends string,
-	ExpectedElement extends ParseSelector<Selector, HTMLElement>,
+	ExpectedElement extends ParseSelector<Selector, HTMLElement | SVGElement>,
```

Comment:
```suggestion
	ExpectedElement extends ParseSelector<Selector, HTMLElement & SVGElement>,
```

Reply:
It fixes the specific error but `HTMLElement & SVGElement` won't let me use anything other than exactly `HTMLElement` or `SVGElement`:

```
[11:34:31 AM] Found 43 errors. Watching for file changes.
```

Example:


```ts
source/features/useful-not-found-page.tsx:206:58 - error TS2345: Argument of type '(description: HTMLDivElement) => Promise<void>' is not assignable to parameter of type 'ObserverListener<HTMLElement & SVGElement>'.
  Types of parameters 'description' and 'element' are incompatible.
    Property 'align' is missing in type 'HTMLElement & SVGElement' but required in type 'HTMLDivElement'.

206  observe('main div[data-testid="eror-404-description"]', showGitObjectHistoryOnRepo, {signal}); // "eror" as misspelled by GitHub
```

Reply:
You may consider function overloads: one for HTMLElement and another for SVGElement.

Reply:
I had forgot to change this type in `getListener` as well, so it was still set to `ExpectedElement extends ParseSelector<Selector, HTMLElement>`


It works now 🎉